### PR TITLE
test(hx-drop-zone): implement component tests

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Test Case `<hx-drop-zone>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2005

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-drop-zone/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+/**
+ * <hx-drop-zone> component tests
+ *
+ * @type HXDropZoneElement
+ *
+ */
+describe('<hx-drop-zone> component tests', () => {
+    const template = '<hx-drop-zone>';
+
+    describe('instantiate element', () => {
```
Comment (100stacks) [inline src/elements/hx-drop-zone/index.spec.js:12]: Please add a test for the public method:

* `drag`

https://github.com/HelixDesignSystem/helix-ui/blob/09e990d69207c176926877731278f751615f401f/src/elements/hx-drop-zone/index.js#L35-L41

Comment (BY00565233) [reply src/elements/hx-drop-zone/index.spec.js:12]: Changes have been done, Please review

File: src/elements/hx-drop-zone/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        });
+    });
+
+    describe('test <hx-drop-zone> getters and setters', () => {
+        it('should be able to drag', async () => {
+            const component = /** @type {HXDropZoneElement} */ await fixture(template);
+            const msg = 'away ';
+
+            component.setAttribute('drag', msg);
+            const dragMsg = component.getAttribute('drag');
```
Comment (100stacks) [inline src/elements/hx-drop-zone/index.spec.js:47]: Try this instead:

```js
const dragMsg = component.drag;
```

Comment (BY00565233) [reply src/elements/hx-drop-zone/index.spec.js]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Pull request merged**